ts/window.d.ts

```diff
@@ -98,5 +100,10 @@ declare global {
     darkTheme: DefaultTheme;
     LokiPushNotificationServer: any;
     LokiPushNotificationServerApi: any;
+
+    getConversationController: () => ConversationController;
```

Comment:
this one has to be removed. 
in typescript you just do `ConversationController.getInstance()` to get the conversation controller